PLUGIN_MIGRATION.md

```diff
@@ -4,108 +4,108 @@
+|  64 | aiven_opensearch_acl_rule                   |        |     2 |
+|  65 | aiven_opensearch_security_plugin_config     |        |     2 |
+|  66 | aiven_opensearch_user                       |        |     2 |
+|  67 | aiven_organization                          | yes    |     2 |
+|  68 | aiven_organization_address                  | yes    |     2 |
+|  69 | aiven_organization_application_user         | yes    |     2 |
+|  70 | aiven_organization_application_user_token   | yes    |     1 |
+|  71 | aiven_organization_billing_group            | yes    |     2 |
+|  72 | aiven_organization_billing_group_list       | yes    |     1 |
+|  73 | aiven_organization_group_project            | yes    |     1 |
+|  74 | aiven_organization_permission               | yes    |     1 |
+|  75 | aiven_organization_project                  | yes    |     2 |
+|  76 | aiven_organization_user                     |        |     2 |
+|  77 | aiven_organization_user_group               |        |     2 |
+|  78 | aiven_organization_user_group_list          | yes    |     1 |
+|  79 | aiven_organization_user_group_member        | yes    |     1 |
+|  80 | aiven_organization_user_group_member_list   | yes    |     1 |
+|  81 | aiven_organization_user_list                | yes    |     1 |
+|  82 | aiven_organization_vpc                      |        |     2 |
+|  83 | aiven_organizational_unit                   | yes    |     2 |
+|  84 | aiven_pg                                    |        |     2 |
+|  85 | aiven_pg_database                           |        |     2 |
+|  86 | aiven_pg_user                               |        |     2 |
+|  87 | aiven_project                               |        |     2 |
+|  88 | aiven_project_user                          |        |     2 |
+|  89 | aiven_project_vpc                           |        |     2 |
+|  90 | aiven_redis                                 |        |     2 |
+|  91 | aiven_redis_user                            |        |     2 |
+|  92 | aiven_service_component                     |        |     1 |
+|  93 | aiven_service_integration                   |        |     2 |
+|  94 | aiven_service_integration_endpoint          |        |     2 |
+|  95 | aiven_service_plan                          | yes    |     1 |
+|  96 | aiven_service_plan_list                     | yes    |     1 |
 |  97 | aiven_static_ip                             |        |     1 |
 |  98 | aiven_thanos                                |        |     2 |
 |  99 | aiven_transit_gateway_vpc_attachment        |        |     2 |
 | 100 | aiven_valkey                                |        |     2 |
 | 101 | aiven_valkey_user                           |        |     2 |
 +-----+---------------------------------------------+--------+-------+
-|     | TOTAL MIGRATED 13%                          | 24     |   178 |
+|     | TOTAL MIGRATED 16%                          | 28     |   178 |
```

Comment:
The `aiven_organizational_unit` PR didn't regenerate the report, because the report was merged right before the unit's PR. Hence, "migrated" counts 2 resources and 2 datasources (13 -> 16%).